### PR TITLE
Parentchain's metadata in enclave

### LINKED ISSUE #27: Get runtime version from metadata
When the service starts, it should get the node's runtime version from the metadata and sends it to the enclave.
So after a runtime upgrade, a restart of the service will fix the inability to send extrinsics.  For now we have to rebuild the code.

### LINKED ISSUE #30: rebase teeracle to worker

### PR DESCRIPTION
**Pull request by echevrier**
- merge worker commit https://github.com/integritee-network/worker/commit/1811bce4e646a6d2148d556f9a62eae516ac3633
- add specific changes for teeracle
- changes coinmarketcap certificate

### CODE REVIEW

**Comment by echevrier** [comment]
Successfully teste with the integritee node. 
Parachain requires an upgrade : https://github.com/integritee-network/parachain/issues/144

File: app-libs/exchange-oracle/src/coin_market_cap.rs [hunk 1]
```diff
@@ -43,8 +43,7 @@ const FIAT_CURRENCY_PARAM: &str = "convert_id";
 const CRYPTO_CURRENCY_PARAM: &str = "id";
 const COINMARKETCAP_PATH: &str = "v2/cryptocurrency/quotes/latest"; // API endpoint to get the exchange rate with a basic API plan (free)
 const COINMARKETCAP_TIMEOUT: Duration = Duration::from_secs(3u64);
-const COINMARKETCAP_ROOT_CERTIFICATE: &str =
-	include_str!("certificates/baltimore_cyber_trust_root_v3.pem");
+const COINMARKETCAP_ROOT_CERTIFICATE: &str = include_str!("certificates/amazon_root_ca_a.pem");
```
Comment (clangenb) [inline app-libs/exchange-oracle/src/coin_market_cap.rs:47]: Why has the root certificate changed here?

**Review by clangenb — APPROVED** [review]
Very nice and quick!

I have one question, but I suppose it is correct! Hence, I approve! :D

File: app-libs/exchange-oracle/src/coin_market_cap.rs (hunk 1 above)
Comment (clangenb) [reply app-libs/exchange-oracle/src/coin_market_cap.rs:47]: Well, this is probably obvious. The old one expired, and for some reason the root authority seems different.

**Pull request merged**